lighthouse-core/test/results/artifacts/artifacts.json

```diff
@@ -861,26 +861,26 @@
         ],
         "description": "Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds",
         "help": "Elements must have sufficient color contrast",
-        "helpUrl": "https://dequeuniversity.com/rules/axe/2.6/color-contrast?application=axeAPI",
+        "helpUrl": "https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI",
         "nodes": [
           {
             "impact": "serious",
             "html": "<h2>Do better web tester page</h2>",
             "target": [
-              "div > h2"
```

Comment:
regression or good thing?

Reply:
> regression or good thing?

I mean, I guess it's accurate, so that's enough? It is the only `h2` in the page. I'm not sure if they recently optimized their selector selection for length or something. Agree they're worse for humans, though.